spotify-artist-viewer/Models/SAArtistViewController.h

```diff
@@ -10,12 +10,7 @@
 @class SAArtist;
 
 @interface SAArtistViewController : UIViewController
-@property (nonatomic)NSString *artistNameVC;
-@property (weak, nonatomic) IBOutlet UIImageView *img;
-@property (weak, nonatomic) IBOutlet UILabel *header;
-@property (weak, nonatomic) IBOutlet UILabel *bio;
-@property (nonatomic) SAArtist *vcArtist;
+@property (strong, nonatomic) SAArtist *vcArtist;
```

Comment:
Just name is property artist. No need to reference the view controller in the name.